django/contrib/postgres/indexes.py

```diff
@@ -43,5 +43,33 @@ def get_sql_create_template_values(self, model, schema_editor, using):
 class GinIndex(Index):
     suffix = 'gin'
 
+    def __init__(self, fields=[], name=None, fastupdate=None, gin_pending_list_limit=None):
+        if gin_pending_list_limit is not None and not (
```

Comment:
Do you see much value in Django validating this? The error message from PostgreSQL seems clear: `django.db.utils.DataError: value 1 out of bounds for option "gin_pending_list_limit"
DETAIL:  Valid values are between "64" and "2147483647".